PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,23 @@
-Please make sure you've read the [pull request](http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) section of our [code contribution guidelines](http://docs.mattermost.com/developer/contribution-guide.html).
+<!--
+Please make sure you've read the pull request section (http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) of our code contribution guidelines (http://docs.mattermost.com/developer/contribution-guide.html).
```

Comment:
The two links outdated. Would you mind just using the text from the server repo here: https://github.com/mattermost/mattermost-server/blob/master/.github/PULL_REQUEST_TEMPLATE.md#L1-L5